projects/ysmart/index.js

```diff
@@ -0,0 +1,14 @@
+const { sumTokensExport } = require("../helper/unwrapLPs");
+
+const masterchef = "0x13C9e00871ce38bA2F11fa0C03E32a9DdD767893";
```

Comment:
this does not look like a smart contract

Reply:
Please see contract on KAVA https://explorer.kava.io/address/0x13C9e00871ce38bA2F11fa0C03E32a9DdD767893/transactions

Reply:
sorry my mistake, I had posted wallet instead of master chef. 
I have corrected address - https://explorer.kava.io/address/0x705A71efadD12faDac67600b666D8c0347848BD3/transactions